docs/configuration.md

```diff
@@ -1002,3 +1004,30 @@ API](http://admin.wechat.com/wiki/index.php?title=Customer_Service_Messages).
 [ to_party: <string> | default = '{{ template "wechat.default.to_party" . }}' ]
 [ to_tag: <string> | default = '{{ template "wechat.default.to_tag" . }}' ]
 ```
+
+## `<telegram_config>`
+```yaml
+# Whether to notify about resolved alerts.
+[ send_resolved: <boolean> | default = true ]
+
+# The Telegram API URL i.e. https://api.telegram.org.
+# If not specified, default API URL will be used.
+[ api_url: <string> | default = global.telegram_api_url ]
+
+# Telegram bot token
+[ bot_token: <string> ]
+
+[ chat_id: <int> ]
+
+# Message template
+[ message: <tmpl_string> default = '{{ template "telegram.default.message" .}}' ]
+
+# Disable telegram notifications
+[ disable_notifications: <boolean> | default = false ]
+
+# Parse mode for telegram messages, can be MarkdownV2 | Markdown | HTML
+[ parse_mode: <string> | default = "MarkdownV2" ]
```

Comment:
In our config documentation convention, is there a way to give a definitive list of alternatives?

Reply:
I added all the possible options, used opsgenie_config priority as reference